src/test/java/org/mastodon/views/trackscheme/bdv/SharedBigDataViewerDataTest.java

```diff
@@ -0,0 +1,33 @@
+package org.mastodon.views.trackscheme.bdv;
+
+import bdv.viewer.ViewerOptions;
+import mpicbg.spim.data.SpimDataException;
+import org.junit.Test;
+import org.mastodon.views.bdv.SharedBigDataViewerData;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+
+public class SharedBigDataViewerDataTest
+{
+	@Test
+	public void testFromSpimDataXmlFileWindowsStyle() throws SpimDataException, IOException
+	{
+		ViewerOptions viewerOptions = new ViewerOptions();
+		SharedBigDataViewerData sharedBigDataViewerData = SharedBigDataViewerData.fromSpimDataXmlFile( "./src/test/resources/org/mastodon/views/trackscheme/bdv/dataset_hdf5.xml", viewerOptions, null );
```

Comment:
In noticed, that there is already a BDV XML+H5 in the resources folder which we could use:
```src/test/resources/org/mastodon/mamut/importer/trackmate/mamutproject/FakeTracksBDV.xml```

Reply:
Thanks for the hint. Made changes to use these resources.